django/http/response.py

```diff
@@ -271,7 +272,10 @@ def make_bytes(self, value):
 
         # If content is already encoded (eg. gzip), assume bytes.
         if self.has_header('Content-Encoding'):
-            return bytes(value)
+            if sys.version_info[0] > 2:
```

Comment:
while this is fine, there are alternatives that make it more clear for the reader. ex: sys.version_info.major